src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -910,10 +941,21 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
         encryptedNote2 = HexStr(ss2.begin(), ss2.end());
     }
 
+    Array arrInputMap;
```

Comment:
[Comment] Maybe some day we should add the standard iteration API to ``json_spirit::Array``? I personally prefer shorter conventional idioms instead of repeated larger code blocks.